samples/Samples.Schemas.Chat/Schema/ChatSchema.cs

```diff
@@ -0,0 +1,11 @@
+namespace GraphQL.Samples.Schemas.Chat;
+
+public class ChatSchema : Schema
```

Comment:
My tests do not use this file, but rather uses `AddAutoSchema` to configure the schema; but the old `Samples.Server.Tests` project does.  I can probably eliminate this file and update the `Samples.Server.Tests` project if you like.  This is the only "code-first" file here; everything else is "type-first", making the sample very easy to read.

Reply:
> I can probably eliminate this file and update the Samples.Server.Tests project if you like.  ... making the sample very easy to read

50/50 here,

Reply:
I'll leave it as-is.